### PR TITLE
Improve security of our donations page

### LINKED ISSUE #179: Ask for user information in the donation flow
Right now we are not asking users for their donation information, we aren't persisting this either. Instead, we are using Stripe as our source of truth.

The goal is to capture more information from users, at least these fields:

- Name
- Email
- Zip Code

These are used by Stripe to validate the transaction.

## Acceptance Criteria

- Ask for user information before processing a donation.
- Send the data asked in the form to Stripe as part of the payment process.

### PR DESCRIPTION
**Pull request by orlando**
**What:** Improving security of our Donations page

**Why:** Close #179 

**How:** 

- Bump Stripe SDK version to **v3**
- Use Stripe Elements instead of our own inputs
- Ask for name, phone and zip code.
- Validate fields before submit.

**Media:**

https://www.loom.com/share/b0c83fe824c747f6a80ed4931a652310

### CODE REVIEW

File: src/javascripts/components/DonationFlow.js [hunk 1]
```diff
@@ -1,41 +1,43 @@
 /* eslint-disable max-len */
 /* global Stripe */
 
+import merge from 'lodash/merge';
 import Widget from '../lib/widget';
 import { postStripePayment } from '../lib/api';
 
+const Checkit = require('checkit');
```
Comment (duranmla) [inline src/javascripts/components/DonationFlow.js]: Why mix `import` and `require`?
If there is no good reason let's just `import`

File: src/javascripts/components/DonationFlow.js [hunk 2]
```diff
@@ -138,25 +142,232 @@ export default class DonationFlow extends Widget {
...
+    const valid = Object.keys(errors).length === 0;
+
+    this.renderErrors(errors);
+
+    return { valid, errors };
+  }
+
+  renderErrors(errors) {
+    // find input
+    Object.keys(errors).forEach(key => {
```
Comment (duranmla) [inline src/javascripts/components/DonationFlow.js]: Don't we have already a problem with the "forEach" on IE?
_Probably were just for Node arrays - can't remember -_

Comment (orlando) [reply src/javascripts/components/DonationFlow.js]: I think you are right, I remember `Array.includes` that was replaced. I'll use `_` for this

Comment (orlando) [reply src/javascripts/components/DonationFlow.js]: Actually we are ok with using `forEach`, it is supported from IE9 up.

**Pull request merged**